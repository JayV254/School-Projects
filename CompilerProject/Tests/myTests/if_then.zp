int main() {

        int a;
        int b;

        a = 10;
        b = 10;

        puts("a is");
        putn(a);
        puts("b is");
        putn(b);

        if(a = b) then {
                puts("a and b are equal!");
		puts("\n");
        }

        return 0;
}

