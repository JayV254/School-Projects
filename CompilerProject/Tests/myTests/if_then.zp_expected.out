a is    10 
b is    10 
a and b are equal!

